apps/common/utils.py

```diff
@@ -78,6 +78,20 @@ def get_all_pages_visible_to_request(request):
     return Page.objects.filter(id__in=pages_for_user.values_list("id", flat=True))
 
 
+def get_next_url_from_request(request):
+    """
+    Get the next URL from the request.
+
+    If the request has a 'next' parameter, then return its value. Otherwise,
```

Comment:
Could we mention that `next` should be populated if being redirected from another url and that HTTP_Referrer is an optional header that gives us the url from where the request originated?